WordPress/Classes/ViewRelated/Suggestions/SuggestionsTableView.h

```diff
@@ -1,7 +1,8 @@
 #import <UIKit/UIKit.h>
 
 typedef NS_CLOSED_ENUM(NSUInteger, SuggestionType) {
-    SuggestionTypeMention
+    SuggestionTypeMentions,
```

Comment:
The change from `SuggestionTypeMention` to `SuggestionTypeMentions` should have been done in https://github.com/wordpress-mobile/WordPress-iOS/pull/15135 since it's not directly related to xposts. Since I already merged that PR I've included it here.